src/main/webapp/chatStyle.css

```diff
@@ -0,0 +1,80 @@
+body {font-family: Arial, Helvetica, sans-serif;}
+* {box-sizing: border-box;}
+
+/* Button used to open the chat form - fixed at the bottom of the page */
+.open-button {
+  background-color: #555;
+  color: white;
+  padding: 16px 20px;
+  border: none;
+  cursor: pointer;
+  opacity: 0.8;
+  position: fixed;
+  top: 23px;
+  right: 28px;
+  width: 280px;
```

Comment:
In general, I would recommend using [relative units](https://www.w3schools.com/cssref/css_units.asp) when defining widths, heights, margins, etc...  instead of pixels because it would allow the website to resize nicely for different window sizes. If you want to keep something fixed with pixels, you can use for example `height: calc(100% - 10px)` (this is a completely random example) to make sure everything else fits on the screen without scrolling.

Reply:
em units should work as well (relative to the font size).